pkg/operator/controller/controller.go

```diff
@@ -406,10 +406,12 @@ func (r *reconciler) ensureDNS(dns *operatorv1.DNS) error {
 		errs = append(errs, fmt.Errorf("failed to create ca bundle configmaps for dns %s: %w", dns.Name, err))
 	}
 
-	cmMap, err := r.caBundleRevisionMap(dns)
-	if err != nil {
-		return fmt.Errorf("failed to generate ca bundle revision map: %w", err)
-	}
+	//cmMap is a map of ca bundle configmaps' names and their resource versions
+	//used in the calculation of the path of ca bundles in Corefile and volume mount path in daemonset.
+	//It can be empty or missing a configmap because it was not found on cluster. In this case,
+	//when a configmap is not found on cmMap, the operator will print a warning, and
+	//it will be ignored while calculating the mentioned paths.
```

Comment:
```suggestion
	// cmMap is a map of ca bundle configmaps' names and their resource versions
	// used in the calculation of the path of ca bundles in Corefile and volume mount path in daemonset.
	// It can be empty or missing a configmap because it was not found on cluster. In this case,
	// when a configmap is not found on cmMap, the operator will print a warning, and
	// it will be ignored while calculating the mentioned paths.
```

Reply:
Can you please supply the default path names that are produced when the cmMap is missing?

Reply:
I've changed the comment a bit. Let me know if it is clearer now. When the cmMap is missing, there will be no path specified for caBundle that will default to using the system certificates. Also, the operator won't add the volume to daemonset. So, it will be safely ignored and the user will be notified through the warnings in the logs. If we decide not to proceed, this will block reconciliation of all the other resources.